src/honeycomb.d.ts

```diff
@@ -58,6 +58,7 @@ export class Grid<T = Hex<T>> extends Array<T> {
     // defined in prototype:
     get(keyOrPoint: number | PointCoordinates): T | undefined
     hexesBetween(firstHex: T, lastHex: T): T[]
+    hexesInRange(centerHex: T, range?: number | 0, includeCenterHex?: | true): T[]
```

Comment:
The ` | 0` in `range?: number | 0` seems redundant, how about just `range?: number`?

And I don't understand the pipe character in `includeCenterHex?: | true`, how about `includeCenterHex?: boolean`?

Reply:
Thanks for the feedback. My intention was to follow your documentation and provide default parameters but just realised that the defaults are already defined in the js function, and optional parameters would suffice. `includeCenterHex` was a mistake all together. I'll follow your suggestion and go with `param?: type`